.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: php
 php:
-  - 5.3
```

Comment:
yeah after some research i find and concluding that php 5.3 are not supported by the unittests as i posted before..

in any case, i tested enough (apart of a already yet fixed dom problem) in php 5.3 production server